app/src/main/java/bubbleheads/buy_cook/TabLayoutFragment.java

```diff
@@ -0,0 +1,47 @@
+package bubbleheads.buy_cook;
+
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class TabLayoutFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater,
```

Comment:
this method is too long. Try to rewrite this having methods not longer than 8-10 lines